scenarios/01_Elensefar.cfg

```diff
@@ -16,12 +16,13 @@
     [story]
         {DOE_PROLOGUE_STORY_OF_SOTA}
     [/story]
+    # you:
     [side]
         controller="human"
         fog=yes
-        {GOLD4 300 300 250 150}
+        {GOLD4 350 300 250 150}
```

Comment:
Giving the easiest level more income to replace lost units makes sense to me, but too much gold at the start just means they're encouraged to spam recruits and have more moves to do before the fighting starts.

Does this help the fun, or does it result in them just paying more upkeep?

Reply:
I mean spamming recruits is usually how I play anyways... if 300 is the absolute maximum, how about lessening the gap between EXPERT and NIGHTMARE and have it be: 
`{GOLD4 300 250 200 150}`
instead?

Reply:
Changed to `{GOLD4 300 250 200 150}` in 3c11649 instead.